staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -315,6 +320,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	ValidatingAdmissionPolicy: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.32
 
+	ConsistentWatchFromEtcd: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
+1

Reply:
Name suggest that it should be enabled by default. Or we make the name negative but then it could be mistaken with a planned feature to serve consistent watch from cache in https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

Reply:
that's why I suggested a name without `Consistent` in it, that described exactly what was being enabled (`WatchFromStorageWithoutResourceVersion`)

Reply:
Done